### PR TITLE
Feat: init supabase

### PR DESCRIPTION
**Pull request by awpala**

### CODE REVIEW

File: services/backend.ts [hunk 1]
```diff
@@ -0,0 +1,359 @@
+import { createClient } from '@supabase/supabase-js';
```
Comment (awpala) [inline services/backend.ts:1]: This file is the bulk of the implementation logic for the GET/READ operations

File: supabase/database.types.ts [hunk 2]
```diff
@@ -0,0 +1,280 @@
+export type Json =
```
Comment (awpala) [inline supabase/database.types.ts:1]: These types are auto-generated via `supabase` CLI, based on table/schema & function definitions set up in Supabase (cloud project)

File: supabase/seed/functions.sql [hunk 3]
```diff
@@ -0,0 +1,272 @@
+/* --- READ OPERATIONS --- */
```
Comment (awpala) [inline supabase/seed/functions.sql:1]: Mirroring definitions here (i.e., `/supabase/seed`) for version control

File: supabase/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Supabase
+
+To generate types dynamically from Supabase, run the following from command line (in location `/website/supabase/`):
+
+```bash
+npx supabase gen types --lang=typescript --project-id <project-id> --schema public > database.types.ts
+```
```
Comment (awpala) [inline supabase/README.md:7]: Preliminary docs, this is the command for generating the types from the schemas/definition in the cloud project locally

**Review by omshub-admin — APPROVED** [review]

**Pull request merged**